dataprofiler/tests/profilers/profiler_options/test_base_option.py

```diff
@@ -1,8 +1,8 @@
 import unittest
 
 from dataprofiler.profilers.profiler_options import BaseOption
-from dataprofiler.tests.profilers.profiler_options.abstract_test_options \
-     import AbstractTestOptions
+from dataprofiler.tests.profilers.profiler_options.abstract_test_options import \
```

Comment:
so this now breaches the 80 char limit, so whats going to happen if both flake8 and isort disagree?